src/storage/txn/scheduler.rs

```diff
@@ -1237,13 +1237,14 @@ impl<E: Engine, L: LockManager> TxnScheduler<E, L> {
                         .map_err(StorageError::from)
                 })
             };
+            let cmd_process_duration = begin_instant.saturating_elapsed();
+            sched_details.cmd_process_nanos = cmd_process_duration.as_nanos() as u64;
             SCHED_PROCESSING_READ_HISTOGRAM_STATIC
```

Comment:
Just curious about it, should it be `SCHED_PROCESSING_WRITE_HISTOGRAM`?

Reply:
`SCHED_PROCESSING_WRITE_HISTOGRAM` represents the `async_write` time, the command processing phase is regarded as the `READ` phase. The name is a little bit confusing.